dolphinscheduler-master/src/main/java/org/apache/dolphinscheduler/server/master/runner/task/DependentTaskProcessor.java

```diff
@@ -173,6 +174,10 @@ private boolean allDependentTaskFinish() {
                 finish = false;
             }
         }
+        if (!finish) {
+            // todo: add information, which dependent doesn't finished
```

Comment:
Maybe there are many unused log when the dependent task is running.
I think we only need to add the log :
1.  all the dependent data 
2.  the finished dependent tasks

Reply:
Done

Reply:
<img width="1014" alt="image" src="https://user-images.githubusercontent.com/22415594/185530974-ac5febc5-d8ea-4fbc-a2a9-1701cb09e8ba.png">